pkg/builtins/range_check.go

```diff
@@ -0,0 +1,84 @@
+package builtins
+
+import (
+	"math"
+
+	"github.com/lambdaclass/cairo-vm.go/pkg/lambdaworks"
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+	"github.com/pkg/errors"
+)
+
+const CHECK_RANGE_BUILTIN_NAME = "range_check"
+const INNER_RC_BOUND_SHIFT = 16
+const INNER_RC_BOUND_MASK = math.MaxUint16
+const CELLS_PER_RANGE_CHECK = 1
+
+const N_PARTS = 8
+
+func RangeCheckError(err error) error {
+	return errors.Wrapf(err, "Range check error")
+}
+
+func OutsideBoundsError(felt lambdaworks.Felt) error {
+	upperBound := uint64(math.Pow(2, float64(N_PARTS*INNER_RC_BOUND_SHIFT)))
+	return RangeCheckError(errors.Errorf("Value %d is out of bounds [0, %d]", felt, upperBound))
```

Comment:
If the idea is to not capitalize the error strings, we should do it everywhere to keep consistency. For the scope of this PR, we can do it only for the lines added.